codec-http/src/main/java/io/netty/handler/codec/http/multipart/InterfaceHttpPostRequestDecoder.java

```diff
@@ -20,7 +20,8 @@
 import java.util.List;
 
 /**
- * This decoder will decode Body and can handle POST BODY (or for PUT, PATCH or OPTIONS).
+ * This decoder will decode Body and can handle POST BODY (or for PUT, PATCH or OPTIONS or any allowed in RFC 7231
+ * except TRACE).
```

Comment:
Ok will propose something more concise
 "(or any methods allowed in RFC 7231, therefore preventing TRACE)"